7-layouts/style.css

```
@@ -0,0 +1,89 @@
+body {
+    color: whitesmoke;
+    margin: 0;
+    font-family: 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
+}
+
+footer {
+    background-color: rgb(39, 39, 39);
+    padding: 6px;
+    text-align: center;
+}
+
+h1 {
+    background-color: rgb(39, 39, 39);
+    font-size: x-large;
+    font-weight: normal;
+    margin: 0;
+    padding: 6px;
+    padding-left: 20px;
+    position: sticky;
+    top: 0;
+}
+h2 {
+    color: aqua;
+    font-size: 10vw;
+    font-weight: 400;
+    margin: 0;
+    text-align: center;
+}
+
+h3 {
+    align-self: flex-end;
+    background-color: rgb(255, 255, 255, 0.25);
```

```suggestion
    background-color: rgb(255, 255, 255, .25);
```
0 is always nothing, so you save 1 character by doing `.25`